docs/Docker-Instructions.md

```diff
@@ -10,8 +10,7 @@ With this feature, you'll be able to:
 
 Because this is a private preview, we'll need your OMS workspace ID to enable it. If you aren't included in the private preview and are interested in joining, drop us a line at OMSContainers@microsoft.com.
 
-## Setting up
-Your container hosts must be running:
+### Supported Linux Operating Systems and Docker:
```

Comment:
It should be supported but we will update more of the Linux OS settings after we tested our matrix list.